azerothcore/azerothcore-wotlk
# Fix(Core/Spell): No visual for Suppression Aura

## Changes Proposed:
-  Changes visual to 0.
-  

## Issues Addressed:

- Partially closes https://github.com/azerothcore/azerothcore-wotlk/issues/11275

## SOURCE:


## Tests Performed:

-  Tested
- 


## How to Test the Changes:


1. Go to suppression room in bwl
2. check if the fire nova effect is gone but the mist affect stays
3.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
